exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/StorageResources.java

```diff
@@ -212,61 +195,15 @@ public Response enablePlugin(@PathParam("name") String name, @PathParam("val") B
   @Consumes(MediaType.APPLICATION_JSON)
   @Produces(MediaType.APPLICATION_JSON)
   public Response updateRefreshToken(@PathParam("name") String name, OAuthTokenContainer tokens) {
```

Comment:
Are these endpoints different in any way from their counterparts in CredentialResources? Could we do everything we need to without the ones here?

Reply:
I had debated removing the endpoints in `StorageResources`.  The only reason I left them was in case someone was using these endpoints, then we'd have a breaking change when they upgraded.   What if I mark them as deprecated?

As a point of clarification, the reason I moved them to `CredentialResources` was because the credentials page allows a user to update their creds.  The `StorageResources` requires admin access, whereas the credentials page only requires an authorized user.

Reply:
Thanks, perhaps deprecation is better even though the next version is 2.0

Reply:
Done!